manifest-build-tools/HWIMO-BUILD

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# HWIMO-BUILD
+#
+# Sets up the virtual environment for the scripts in question
+# Runs the specified command, passing down all command line arguments
+#
+
+# Specify full error checking
+set -e
+
+function cleanup {
+  set +e
+  trap '' EXIT SIGINT
```

Comment:
The trap command means to execute a command when a signal is received.
```trap command signals```
But the command here means doing nothing if received "EXIT" or "SIGINT" sinnal.
I think the command is useless here. But I'm not sure.

I tried to remove the line and the script can run successfully.
And I read other examples of clean up function.
I didn't find any example which run  trap command in clean up function~